timer-8018x.c

```diff
@@ -5,7 +5,199 @@
 #include "emu-int.h"
 #include "emu-timer.h"
 
+#include "emu-int.h"
 #include "int-8018x.h"
 #include "timer-8018x.h"
 
-#include "timer-8018x.inc"
+#include <stdio.h>
```

Comment:
Should `timer-8018x.inc` be wiped out ?

Reply:
Could you please post the 'EB' datasheet that you are working with ? I will compare it with the R8810 one to see if both timer implementations could be merged.

Reply:
Will try to merge the implementations later.